e2e/test/scenarios/embedding/embedding-smoketests.cy.spec.js

```diff
@@ -310,7 +309,6 @@ describe("scenarios > embedding > smoke tests", { tags: "@OSS" }, () => {
 });
 
 function resetEmbedding() {
-  cy.request("PUT", "/api/setting/enable-embedding", { value: false });
```

Comment:
this line can be removed because `enable-embedding` was true in the snapshot, but our new setting is `false`, so we don't need to reset it. On the other hand, we need to set it to `true` for tests that we didn't reset it previously.